src/main/java/com/openshift/restclient/ClientBuilder.java

```diff
@@ -183,7 +189,9 @@ public IClient build() {
 			dispatcher.setMaxRequests(maxRequests);
 			dispatcher.setMaxRequestsPerHost(maxRequestsPerHost);
 
-			OkHttpClient.Builder builder = new OkHttpClient.Builder()
+			OkHttpClient.Builder builder = null;
+			if (!this.sslCertCallbackWithDefaultHostnameVerifier) {
```

Comment:
I'm missing the point of the change here.  It looks like the only difference is to alter the read and write timouts  if we turn off the default hostname verifier.  I don't see the relation to between verification and read/writes on the channel.  We should be building the client with the read/write timeouts regardless